data/dineshlalam15.json

```
@@ -0,0 +1,18 @@
+{
+  "name": "Dinesh Kumar Lalam (test)"
```

There is a missing comma `,`

```suggestion
  "name": "Dinesh Kumar Lalam (test)",
```